content/docs/ref/pr.md

```diff
@@ -17,12 +17,11 @@ preventing an infinite chain of runs.
 
 Any [generic option](/doc/ref) in addition to:
 
-```
---md          Output in markdown format [](url).                     [boolean]
---remote      Sets git remote.                    [string] [default: "origin"]
---user-email  Sets git user email.   [string] [default: "olivaw@iterative.ai"]
---user-name   Sets git user name.            [string] [default: "Olivaw[bot]"]
-```
+- `--md`: Produce output in markdown format (`[Pull/Merge Request](url)`).
```

Comment:
`cml pr` has a `--md` flag but no `-f, --file` flag. We should either add one, or remove `-f, --file` from `cml publish` & `cml tensorboard-dev`

Reply:
Maybe add an example of the output of `cml pr --md` to show how it is different from plain `cml pr`? 

It doesn't seem intuitive to me what the difference would be without an example:

https://github.com/iterative/cml/blob/master/src/cml.js#L339-L343

Reply:
Unless `-f` does some extra logic that you can't achieve with plain output redirection, I would drop `--file` options from other commands (maybe adding example instead like `cml cmd --md > output.md`)

Reply:
removed